copyspecial.py

```diff
@@ -6,18 +6,56 @@
 # Google's Python Class
 # http://code.google.com/edu/languages/google-python-class/
 
-import sys
+
 import re
 import os
 import shutil
 import commands
```

Comment:
FYI: commands module was deprecated in Python 3. It's now the subprocess module.